release-content/0.14/release-notes/12910_Implement_percentagecloser_filtering_PCF_for_point_lights.md

```diff
@@ -1 +1,13 @@
-TODO
+Percentage-closer filtering is a standard anti-aliasing technique used to get softer, less jagged shadows.
+To do so, we sample from the shadow map near the pixel of interest using a Gaussian kernel, averaging the results to reduce sudden transitions as we move in / out of the shadow.
+
+As a result, Bevy's point lights now  look softer and more natural, without any changes to end user code. As before, you can configure the exact strategy used to anti-alias your shadows by setting the [`ShadowFilteringMethod`](https://dev-docs.bevyengine.org/bevy/pbr/enum.ShadowFilteringMethod.html) component on your 3D cameras.
+
+{{ compare_slider(
+    left_title="Without PCF filtering",
+    left_image="before_pcf.png",
+    right_title="With PCF filtering",
+    right_image="after_pcf.png"
+) }}
```

Comment:
This slider does not appear to work. Is there an existing issue about this?

![image](https://github.com/bevyengine/bevy-website/assets/59022059/77840031-1fff-4834-806d-811c0e9c4f32)

Reply:
Ugh, okay. No existing issue. I may have simply screwed this up: I copied it from the 0.13 release notes.

Reply:
I'm going to merge without fixing this: https://github.com/bevyengine/bevy-website/issues/1424 is quite involved.